src/app/pages/component-viewer/component-viewer.ts

```diff
@@ -88,7 +89,7 @@ export class ComponentBaseView implements OnInit, OnDestroy {
   ngOnInit() {
     this.componentViewer.componentDocItem.pipe(takeUntil(this.destroyed)).subscribe(() => {
       // 100ms timeout is used to allow the page to settle before moving focus for screen readers.
-      setTimeout(() => this.focusTarget.nativeElement.focus({preventScroll: true}), 100);
+      setTimeout(() => this.focusTarget.nativeElement.id = 'main-content', 100);
```

Comment:
We probably don't need the setTimeout since we're not triggering a focus anymore, should be fine to set the id immediately

Reply:
I was able to just remove this entirely because now `main-content` is set to the the tab headers.